pkg/meta/repodb/storage_parsing.go

```diff
@@ -118,15 +109,18 @@ func ParseRepo(repo string, repoDB RepoDB, storeController storage.StoreControll
 		}
 
 		if isSignature {
-			// We'll ignore signatures now because the order in which the signed image and signature are added into
-			// the DB matters. First we add the normal images then the signatures
-			signaturesFound = append(signaturesFound, foundSignatureData{
-				repo:                 repo,
-				tag:                  tag,
-				signatureType:        signatureType,
-				signedManifestDigest: signedManifestDigest.String(),
-				signatureDigest:      digest.String(),
-			})
+			err := repoDB.AddManifestSignature(repo, digest,
```

Comment:
Does it matter if the signature is added before or after the image it is signing? The old code added the signatures after.

Reply:
With this PR the order doesn't matter anymore.